Hush/Assets/Scripts/Relics/RelicEnemySpawner.cs

```diff
@@ -18,14 +20,27 @@ void SpawnEnemies()
 
          foreach (var spawn in spawnPoints)
          {
-            var enemyClone = Instantiate(enemyPrefab, spawn.transform.position , Quaternion.identity);
-         
-            var target = GameObject.FindWithTag(Tags.Relic);
-            enemyClone.transform.LookAt(target.transform, Vector3.up);
-         
-            //TODO change clone vision
+            Vector3 position = spawn.transform.position;
+            
+            CreateSmoke(position);
+            StartCoroutine(CreateEnemy(position));
          }
       }
-   
+
+      private void CreateSmoke(Vector3 position)
+      {
+         var smokeClone = Instantiate(smokePrefab, position, Quaternion.identity);
+         smokeClone.GetComponent<ParticleSystem>().Play();
+         Destroy(smokeClone, 1.5f);
+      }
+
+      IEnumerator CreateEnemy(Vector3 position)
+      {
+         yield return new WaitForSeconds(.3f);
+         
+         var enemyClone = Instantiate(enemyPrefab, position, Quaternion.identity);
+         var target = GameObject.FindWithTag(Tags.Relic);
```

Comment:
Don't need to find the target every time you spawn an enemy... should either be a serializeField or Find 1x and then set this